tests/dragonfly/cluster_test.py

```diff
@@ -3025,6 +3025,53 @@ async def wait_for_errors_num(client, err_num, timeout=10):
     await wait_for_errors_num(c_nodes[0], 2)
 
 
+@pytest.mark.slow
+@pytest.mark.exclude_epoll
+@pytest.mark.asyncio
+@dfly_args({"cluster_mode": "yes"})
+async def test_migration_to_slow_target_node(df_factory: DflyInstanceFactory, df_seeder_factory):
+    # 1. Create cluster of 3 nodes with all slots allocated to first node.
+    instances = [
+        df_factory.create(
+            port=next(next_port),
+            admin_port=next(next_port),
+            proactor_threads=16 if i == 0 else 1,
+            vmodule="outgoing_slot_migration=2,cluster_family=2,incoming_slot_migration=2,streamer=2",
+        )
+        for i in range(2)
+    ]
+    df_factory.start_all(instances)
+
+    nodes = [(await create_node_info(instance)) for instance in instances]
+    nodes[0].slots = [(0, 9000)]
+    nodes[1].slots = [(9001, 16383)]
+
+    await push_config(json.dumps(generate_config(nodes)), [node.admin_client for node in nodes])
+
+    logging.debug(f"DEBUG POPULATE")
+    await nodes[0].client.execute_command("DEBUG POPULATE 100000 test 100 RAND SLOTS 0 9000")
+
+    logging.debug("start seeding")
+    seeder = df_seeder_factory.create(
+        keys=50_000, port=instances[0].port, cluster_mode=True, pipeline=False
+    )
+    await seeder.run(target_deviation=0.1)
```

Comment:
I dont think you need this step , you can just start running seeder without filling the data first as you already run the debug populate